### ISSUE TITLE
ChatGLM-6B full parameter fine-tuning loss = 0

### ISSUE THREAD

**Issue by xiaojunjun65** [shibing624/textgen#44]
Today, when I was doing the full parameter fine-tuning, my loss became 0.0 after the training of batch 17, and then it kept going to 0.0. I changed both the learning rate and batch, but I still failed to solve such problems

![1686814220592](https://github.com/shibing624/textgen/assets/105632434/a2e9fb57-7147-4773-b76e-6f8729e11d9a)

![c48c3a4aade14a25914550b827a0c3f](https://github.com/shibing624/textgen/assets/105632434/ac65738d-a305-426b-a2cd-5b58050a4cc9)

**Comment by shibing624** [comment]
try to set float32, not fp16.
i run full parameter fine-tuning success with llama, not chatglm ever.
